src/context/canvas.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2017-present Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the MIT License (the "License"); you may obtain a copy of the
+ * license at https://github.com/palantir/svg-typewriter/blob/develop/LICENSE
+ */
+
+import { IAnchor, ITransform } from "../writers";
+import { ITypesetterContext } from "./index";
+
+/**
+ * Options for styling text
+ */
+export interface ICanvasFontStyle {
```

Comment:
in SVG it's entirely controlled by CSS.

We could add some optional common params that can be applied directly to the tag like `<text style="font: 10px arial">`